code/modules/reagents/reagent_containers/chem_heirloom.dm

```diff
@@ -0,0 +1,48 @@
+//Chemist's heirloom
+
+/obj/item/reagent_containers/glass/chem_heirloom
+	volume = 100
```

Comment:
I think the volume should be 0 until the round end, so you can't put stuff into this either